src/app/settings/settings.component.ts

```diff
@@ -48,6 +55,8 @@ export class SettingsComponent extends RouterEnter {
     // get contact number and email from local storage
     this.profile.email = this.storage.getUser().email;
     this.profile.contactNumber = this.storage.getUser().contactNumber;
+    this.profile.image = this.storage.getUser().image;
```

Comment:
Put a path of default image if no data passed in

Reply:
zaw done some changes, back end return image url always. if there are no image back end will send gravator url with default image added. so no need to put that from front end.

Reply:
https://my.practera.com/img/user-512.png this url is the default image url this always added to image url back end returns.